src/bin/collapse-guess.rs

```diff
@@ -23,10 +29,36 @@ struct Opt {
     #[structopt(short = "v", long = "verbose", parse(from_occurrences))]
     verbose: usize,
 
+    /// Number of stacks per job sent to threadpool (only used if nthreads > 1)
```

Comment:
Same here. I'm leaning towards not exposing either of these on the CLI.

Reply:
Yah - I agree we should stop exposing `nstacks` to the user.